integration-tests/chopsticks/src/network/utils.ts

```diff
@@ -0,0 +1,53 @@
+export function getSiblingDestination(paraId: number) {
```

Comment:
Nitpick, but I would call this `getSiblingLocation`, or `getSiblingMultiLocation`.

Reply:
Same for the others.

Reply:
update in: [41c2859](https://github.com/KILTprotocol/kilt-node/pull/614/commits/41c2859fbd6ce083c9dc7fc8ea53a50bc36fb37a)